xbmc/settings/AdvancedSettings.cpp

```diff
@@ -576,6 +578,7 @@ void CAdvancedSettings::ParseSettingsFile(const std::string &file)
   pElement = pRootElement->FirstChildElement("video");
   if (pElement)
   {
+    XMLUtils::GetInt(pElement, "subtitleverticalmargin", m_videoSubtitleVerticalMargin);
```

Comment:
Needed? We should avoid advanced settings whenever possible

Reply:
has been requested in the forum and i find this extra functionality very useful,
because it allows you to move the hardcoded vertical margin to all types of subtitle position,
if no one is against, i would prefer to keep this feature, is minimally invasive

Reply:
Sure, it's okay to me. The reason I asked is because it misses documentation and as such I missed your original intention. Please describe the setting if possible near the variable declaration. It will also help filling the wiki later